src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -1267,20 +1269,18 @@ public void SetAvailableOrientations(List<Window.WindowOrientation> orientations
         }
 
         /// <summary>
-        /// Sets window position and size for specific orientation.
-        /// This api reserves the position and size per orientation to display server.
-        /// When the device is rotated, the window is moved/resized with the reserved position/size by display server.
-        /// Currently, it only works when the window's type is WindowType::Ime.
-        /// It means this function is only for IME window of internal keyboard application.
-        /// It is only for internal keyboard application.
-        /// This should be hidden.
+        /// Reserves IME window's size for specific orientation to Tizen display server.
+        /// It only works special window that has IME WindowType.
+        /// If application uses this method to reserve window's size, WindowSize, WindowPosition and WindowPositionSize don't work.
+        /// In addition, the window's position is set by Tizen display server, if application uses this method.
         /// </summary>
-        /// <param name="positionSize">The reserved position and size for the orientation.</param>
+        /// <param name="size">The reserved size for the orientation.</param>
         /// <param name="orientation">The orientation.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public void SetPositionSizeWithOrientation(Rectangle positionSize, Window.WindowOrientation orientation)
+        /// <since_tizen> 9 </since_tizen>
+        public void ReserveSizePerOrientationForIme(Size2D size, Window.WindowOrientation orientation)
```

Comment:
can you share how efl provides the public interface?
i wish it can be provided in separated file .

Reply:
EFL interface only has "void ecore_wl2_window_rotation_geometry_set(EcoreWindow window, int angle, int x, int y, int w, int h).
This function only works for "IME" window surface.
In addition, input params as x and y don't have any effect. 
Because the IME window position is decided by display server.